examples/src/main/java/ExampleViewWebhook.java

```diff
@@ -22,11 +22,11 @@ public static void main(String[] args) {
 
         try {
             System.out.println("Viewing webhook..");
-            final String webHookId = args[1];
-            //Viewing webHook by webHook id
-            final WebhookResponseData webHookResponseDataList = messageBirdClient.viewWebHook(webHookId);
-            //Display WebHook Response Data
-            System.out.println(webHookResponseDataList.toString());
+            final String webhookId = args[1];
```

Comment:
please write an example class for updateWebhook.

Reply:
The library contains a lot of examples already and users have almost zero insights from it. I think adding an example for every REST call is overkill :)

Reply:
Okay, I saw you also don't add an example for listwebhook, I thought at least updatewebhook can be written.

Reply:
Let’s not write examples just for the sake of writing examples. Most of the examples contain single REST call and extra argument parsing, not mentioning amount of duplicated code. Again, I don’t see how users could benefit from such examples.